plugins/tileacq.py

```diff
@@ -655,6 +655,7 @@ def acquire(self, dlg):
         main_data = self.main_app.main_data
         str_ctrl = main_data.tab.value.streambar_controller
         stream_paused = str_ctrl.pauseStreams()
+        dlg.pauseSettings()
```

Comment:
Also need to call `resumeSettings()`. Probably line 764 (just after `dlg.Destroy()`) and in the `CancelledError` handler. And you need to change the `return`s from the cancellation place to `raise CancelledError`.